sdf/bezier.go

```diff
@@ -378,21 +388,21 @@ func (v *BezierVertex) Mid() *BezierVertex {
 }
 
 // HandleFwd sets the slope handle in the forward direction.
-func (v *BezierVertex) HandleFwd(theta, r float64) *BezierVertex {
+func (v *BezierVertex) HandleFwd(theta, r float64) (*BezierVertex, error) {
```

Comment:
You'll notice that this function is vertex in, vertex out. That lets you write things like:

x.func1().func2().func3()

Which is a nice property when it comes to building a node/vertex and decorating it with various properties.
Adding an error return to it stops you from doing this. So in this instance I would say concise code beats error handling. ie - leave it as a panic.

Reply:
Yeah, I was also concerned that the method chaining ability would get removed by some of these changes -- we talked about this amongst ourselves as well, and I used the .Rel() and .Polar() examples to demonstrate it to folks.  

A fair compromise that I can see might be to have the core method return an err and then have a wrapper method panic if it gets that err.  That way callers can choose the behavior they need.

Reply:
- Reverted these in 9d211c8
- I'll keep the the split-level err/wrapper idea in mind for later if/when it's needed for application code.